core/test/acceptance/admin/users_spec.js

```diff
@@ -329,5 +329,10 @@ describe('User API', function () {
                 should.exist(res.body.password);
                 should.exist(res.body.password[0].message);
             });
+
+        await request
```

Comment:
Even if we stick with this method - we have a test! so we won't accidentally break it